server/localization.go

```diff
@@ -155,6 +155,17 @@ var Locales = []*Locale{
 			"Forum":    "Foro",
 		},
 	},
+	{
+		Subdomain: "jp",
+		Image:     "static/images/locales/jp.png",
+		Translations: map[string]string{
+			"Activity": "活動",
+			"Thread":   "糸",
+			"Poster":   "著者",
+			"Time":     "時間",
```

Comment:
Some of these are too literal. These translations would be more natural:

```suggestion
			"Activity": "アクティビティ",
			"Thread":   "スレッド",
			"Poster":   "投稿者",
			"Time":     "日時",
```